metadata-integration/java/spark-lineage/src/main/java/datahub/spark/model/dataset/JdbcDataset.java

```diff
@@ -1,19 +1,29 @@
 package datahub.spark.model.dataset;
 
+import java.util.HashMap;
+import java.util.Map;
+
 import com.linkedin.common.FabricType;
-import com.linkedin.common.urn.DataPlatformUrn;
 import com.linkedin.common.urn.DatasetUrn;
 
+import datahub.spark.model.LineageUtils;
+import io.opentracing.contrib.jdbc.parser.URLParser;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
 
 @EqualsAndHashCode
 @ToString
 public class JdbcDataset implements SparkDataset {
+  //TODO: Should map to the central location on datahub for platform names
+  private static final Map<String, String> PLATFORM_NAME_MAPPING = new HashMap<>();
+  static {
+    PLATFORM_NAME_MAPPING.put("postgresql", "postgres");
```

Comment:
why not add more mappings here?

Reply:
For others url to platform name is same hence have not added them there.